lib/Controller/PageController.php

```diff
@@ -128,11 +134,12 @@ protected function guestEnterRoom($roomId) {
 			throw new HintException($this->l10n->t('The room does not exist.'));
 		}
 
+		$newSessionId = $this->secureRandom->generate(255);
 		$params = [
-			'sessionId' => 'lukas-fix-rand',
+			'sessionId' => $newSessionId,
 			'roomId' => $roomId,
 		];
-		$response = new TemplateResponse($this->appName, 'index', $params, 'blank');
+		$response = new TemplateResponse($this->appName, 'index-public', $params, 'base');
```

Comment:
@jancborchardt Could you please take a look to the css of the public page? It only shows the content correctly when you make window width smaller.